java/code/src/com/suse/manager/webui/controllers/contentmanagement/handlers/ValidationUtils.java

```diff
@@ -31,7 +29,8 @@ private ValidationUtils() { }
      * @return true if label is valid
      */
     public static Boolean isLabelValid(String label) {
-        return Pattern.compile(PROJECT_LABEL_REGEX).matcher(label).find();
+        return Pattern.compile(com.redhat.rhn.manager.channel.CreateChannelCommand.CHANNEL_NAME_REGEX)
```

Comment:
I guess this was supposed to be `CHANNEL_LABEL_REGEX` instread, right?

Reply:
Actually, it should be both :laughing: The bug was exactly that!

the project label will be part of both the cloned channel label and name!

Reply:
But in this case we should use the more restrictive regex - IIUC the name regex allows also spaces. Am I missing something?

Reply:
Yes, you are right. But the label regex accept numbers and the name regex doesn’t, I guess in this case we have to validate with both. I will add that :)

Reply:
I adapted the validation rule to include both Regex :) It should be good now